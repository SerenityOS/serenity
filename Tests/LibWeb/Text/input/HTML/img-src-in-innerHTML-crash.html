<script src="../include.js"></script>
<script>
    test(() => {
        let e = document.createElement("div");
        e.innerHTML = "<img src='data:text/png,'>";
        println("PASS! (Didn't crash)");
        println(e.firstChild.src);
    });
</script>
