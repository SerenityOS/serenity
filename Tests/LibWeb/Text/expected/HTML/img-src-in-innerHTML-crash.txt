PASS! (Didn't crash)
data:text/png,
